pom.xml

```diff
@@ -81,6 +81,16 @@
             <artifactId>spring-boot-autoconfigure-processor</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-configuration-processor</artifactId>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
```

Comment:
Waar is deze voor? En moet deze dep niet ook worden ge-exclude in de spring boot maven plugin?

Reply:
was een test om te kijken of het iets deed. Deze kan weg